frontend/public/components/monitoring/alert-manager-config.tsx

```diff
@@ -97,7 +270,25 @@ type AlertManagerConfigurationProps = {
   onCancel?: () => void;
 };
 
-type AlertRoutingProps = {
+type AlertManagerProps = {
   config: any;
-  secret: K8sResourceKind;
+  secret?: K8sResourceKind;
+};
+
+type ReceiverTableProps = {
+  data: any;
+  filterValue?: string;
+  route: any;
+};
+
+type ReceiverTableRowProps = {
+  obj: any;
```

Comment:
Can some of these `any` types be replaced with something more specific? At least specifying that they are objects with `{ [key: string]: string }` or similar? There is also `unknown`, which is useful in some cases.

Reply:
Replaced all 'any`'s with more specific types.